torch_xla/csrc/tensor_util.cpp

```diff
@@ -337,8 +338,18 @@ at::Tensor MakeTensorFromXlaLiteral(const xla::Literal& literal) {
           literal, at::ScalarType::Short, xla::PrimitiveType::S16);
     }
     case xla::PrimitiveType::S32: {
-      return XlaLiteralToTensor<xla::int32, int32_t>(
-          literal, at::ScalarType::Int, xla::PrimitiveType::S32);
+      switch (dest_element_type) {
```

Comment:
Given that we support tensor rewriting at this point, this should be fully (or almost fully) populated.

Reply:
What do you mean?